lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -2646,6 +2596,8 @@ namespace {
         }
       }
 
+      // FIXME: Subscript?
```

Comment:
we don't support distributed subscripts (thankfully), so can remove the fixme I think?

Reply:
Looking at the line below I think we just handle it via fallthrough into the general case:

```swift
// This is either non-distributed variable, subscript, or something else.
       ctx.Diags.diagnose(declLoc,
                          diag::distributed_actor_isolated_non_self_reference,
```

so nothing to do here for the fixme?

Reply:
I didn't change anything with respect to distributed subscripts, but wanted to note that this is where we'd deal with them if/when we do